mw/diag/dtc/database/dtc_database.cpp

```diff
@@ -21,7 +21,7 @@ core::ErrorCode DtcDatabase::ManResetError(uint16_t error_id){
     if (error==this->errors_.end()){
         return core::ErrorCode::kNotDefine;
     }
-    error->second.SetStatus(DtcErrorStatus_t::kManReset);
+    error->second.SetStatus(0x2);
```

Comment:
można by było opisać czemu 0x02 i to 0x01 było by czytelniej ale to tylko tak informacyjnie nie czepiam się do tego

Reply:
myślałem żeby dać tu enuma ale nwm czy to dobry pomysł

Reply:
sam jeszcze nie wiem czemu tam jest 0x2

Reply:
ok fajnie jak byś wiedział ;)